src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/IInstanceRepository.cs

```diff
@@ -19,12 +19,12 @@ public interface IInstanceRepository
         Task<List<Instance>> GetInstancesOfInstanceOwner(int instanceOwnerId);
 
         /// <summary>
-        /// Gets all the instances for an instance owner
+        /// Gets all the instances for an instance owner that should be visible for the user and in a given state
         /// </summary>
         /// <param name="instanceOwnerId">the owner of the instance</param>
-        /// <param name="instanceState">instance state. Valid values_ active, archived, deleted</param>
+        /// <param name="instanceState">instance state. Valid values: active, archived, deleted</param>
         /// <returns>The list of instances for an instance owner</returns>
-        Task<List<Instance>> GetInstancesInStateOfInstanceOwner(int instanceOwnerId, string instanceState);
+        Task<List<Instance>> GetVisibleInstancesInStateOfInstanceOwner(int instanceOwnerId, string instanceState);
```

Comment:
I don't think the method name must be changed. VisibleDateTime is just a parameter and going further we will include more parameters for filtering, So i think this name change is unnecessary